src/Home/Footer/style.module.css

```diff
@@ -0,0 +1,39 @@
+.footer {
+  width: 100%;
+  text-align: left;
+  padding: var(--spacing-1) 0;
```

Comment:
where are these variables being defined? How are you importing them?

How about we have a utils.css file which contains all the CSS variables, and then import them wherever needed.
We can have separate CSS files like colors.css, sizes.css etc, to make sure it scales well.

Reply:
I defined them in the index.css file. I have now moved them to separate files and imported them in a utils.css file 
Thanks for the feedback